internal/controller/api/v1alpha1/cluster_types.go

```diff
@@ -60,7 +60,7 @@ type ClusterProgressInfo struct {
 
 type HostNetwork struct {
 	// cidr
-	Cidr string `json:"cidr,omitempty"`
+	CIDR string `json:"CIDR,omitempty"`
```

Comment:
For isolated acronyms, the conventions and examples support the json form being all lower-case. Again looking at the Volume resource, it has fields such as `nfs`, `iscsi`, `rbd`, etc.

Reply:
Ack. that also applies to `cpu` and `mtu` (and more) in the WIP host CRD. will take that into account